src/scraper/spiders/cpp_symbol_spider.py

```diff
@@ -164,8 +186,8 @@ def parse_type(resp: scrapy.http.Response):
         sigs = get_signatures(resp)
         desc = resp.css("div.mw-content-ltr").xpath("string(p)").extract()
         # resp.css("table.t-dsc-begin").xpath("string(tr)").extract()
-        types = ["P L A C E H O L D E R"]
-        funcs = ["P L A C E H O L D E R"]
+        types = get_from_table("Member type", resp.css("table.t-dsc-begin").extract())
+        funcs = get_from_table("member function", resp.css("table.t-dsc-begin").extract())
```

Comment:
If you use ``resp.css("table.t-dsc-begin").extract()`` twice, you might as well assign it to a variable. Considering this code runs a dozen times each extraction, that could save some time.